pkg/api/controller.go

```diff
@@ -3651,6 +3652,19 @@ func (c *Controller) GetLakeFSVersion(w http.ResponseWriter, r *http.Request) {
 	})
 }
 
+func (c *Controller) GetGarbageCollectionConfig(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
+	_, err := auth.GetUser(ctx)
+	if err != nil {
+		writeError(w, r, http.StatusUnauthorized, ErrAuthenticatingRequest)
+		return
+	}
+
+	writeResponse(w, r, http.StatusOK, GarbageCollectionConfig{
+		GracePeriod: aws.Int(int(ref.LinkAddressTime)),
```

Comment:
Can you add lakeFS version as well

Reply:
You have `/config/version` endpoint for that.